muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/workspaces/index.ts

```diff
@@ -14,6 +14,7 @@ import {
 } from '~/reducers/workspaces';
 import equals = require("deep-equal");
 import $ from '~/lib/jquery';
+import { getJSDocDeprecatedTag } from 'typescript';
```

Comment:
This is not good, added most of the typescript transpiler source to the build!...